packages/preset-attributify/src/variants/pseudo.ts

```diff
@@ -0,0 +1,37 @@
+import type { CSSEntries, VariantObject } from '@unocss/core'
+import { CONTROL_BYPASS_PSEUDO_CLASS, PseudoClasses } from '@unocss/preset-mini/variants'
```

Comment:
I kinda want to avoid relying preset-mini on attributify (as it expects to empower any rules to attributify). I think it might be better to have this logic somehow in the preset-mini instead.

Reply:
Something like this? https://github.com/antfu/unocss/pull/268/commits/2ddc566d99b9d63f3c64210d44754a7c7b449cd9

Reply:
Getting ahead of myself, the spec says `:not`, `:is` and `:where` have similar syntax, so you can generalize NotRE as such:
https://github.com/chu121su12/unocss/compare/preset-attibutify-variant-2...chu121su12:pseudo-class-fn

https://developer.mozilla.org/en-US/docs/Web/CSS/:not#syntax
https://developer.mozilla.org/en-US/docs/Web/CSS/:where#syntax
https://developer.mozilla.org/en-US/docs/Web/CSS/:is#syntax

`:has` can be added too, but there's no browser support yet
https://developer.mozilla.org/en-US/docs/Web/CSS/:has#syntax